Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -119,6 +119,17 @@ class CartViewModel: BaseViewModel {
         return cart.value?.lineItems?[indexPath.row].totalPrice?.description ?? "N/A"
     }
 
+    func productDetailsViewModelForLineItemAtIndexPath(indexPath: NSIndexPath) -> ProductViewModel {
+        var product = ProductProjection()
+        if let lineItem = cart.value?.lineItems?[indexPath.row] {
+            product.id = lineItem.productId
+            // When navigating to PDP from LineItem, we only show one variant
+            product.masterVariant = lineItem.variant
+            product.name = lineItem.name
```

Comment:
I appreciate the idea, because it'll have one less API call. However, if e.g. the customer decides that he wants to add the same product in a different size or color, it won't work from this view. I'd feel better if we grab the full product from the API in this case.
WDYT?